packages/embark/src/cmd/cmd_controller.js

```diff
@@ -27,7 +28,7 @@ class EmbarkController {
     this.events = new Events();
     this.logger = new Logger({logLevel: Logger.logLevels.debug, events: this.events, context: this.context});
 
-    this.config = new Config({env: env, logger: this.logger, events: this.events, context: this.context, version: this.version});
+    this.config = new Config({env: env, logger: this.logger, events: this.events, context: this.context, version: this.version, pkg: PKG});
```

Comment:
I would personally prefer naming it `package`

Reply:
agree